node:internal/process/promises:288
            triggerUncaughtException(err, true /* fromPromise */);
            ^

[Error: ENOENT: no such file or directory, open 'test/data/input/testfun-scop1.calc'] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: 'test/data/input/testfun-scop1.calc'
}

Node.js v18.8.0
